sonar-db/src/main/java/org/sonar/db/ce/CeScannerContextMapper.java

```diff
@@ -17,8 +17,12 @@
  * along with this program; if not, write to the Free Software Foundation,
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
-@ParametersAreNonnullByDefault
-package org.sonar.db.scannercontext;
+package org.sonar.db.ce;
 
-import javax.annotation.ParametersAreNonnullByDefault;
+import java.util.List;
+import org.apache.ibatis.annotations.Param;
 
+public interface CeScannerContextMapper {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Annotate the "CeScannerContextMapper" interface with the @FunctionalInterface annotation [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1609)